src/components/accordion/daikin-accordion.ts

```diff
@@ -0,0 +1,199 @@
+import { cva } from "class-variance-authority";
+import { LitElement, css, html, unsafeCSS } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import tailwindStyles from "../../tailwind.css?inline";
+
+const cvaDetails = cva(
+  [
+    "flex",
+    "w-full",
+    "h-max",
+    "bg-white",
+    "font-daikinSerif",
+    "overflow-clip",
+
+    "[&>summary]:open:border-b",
+    "[&>summary]:open:border-b-[#CECECE]",
+
+    "[&>summary:hover]:[&[aria-disabled='false']]:bg-[#DCDCDC]",
+    "[&>summary:hover]:[&[aria-disabled='false']]:cursor-pointer",
+    "[&>summary:focus-visible]:[&[aria-disabled='false']]:outline",
+    "[&>summary:focus-visible]:[&[aria-disabled='false']]:outline-[3px]",
+    "[&>summary:focus-visible]:[&[aria-disabled='false']]:outline-[#0097E0]",
+    "[&>summary:focus-visible]:[&[aria-disabled='false']]:outline-offset-[-3px]",
+
+    "[&>summary:after]:[&[data-open]]:rotate-180",
+  ],
+  {
+    variants: {
+      disabled: {
```

Comment:
status might be better.

```suggestion
      status: {
```